dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -292,6 +293,8 @@ TaskFunction createStackSizeTest() {
         '--driver', testDriver,
         '-d',
         deviceId,
+        '--screenshot',
+        hostAgent.dumpDirectory.path,
```

Comment:
I ran into issue where local environment does not have FLUTTER_LOGS_DIR

Reply:
That's a regression from https://github.com/flutter/flutter/pull/84008. Fix out at https://github.com/flutter/flutter/pull/84820